### DISCUSSION TITLE
[Frontend] barrel export 변경해야할까요?

### DISCUSSION THREAD

**Discussion by dmdgpdi** [Kernel360/F2-BAGUNI#188] [개발 토의]
[[Korean FE Article] 웹 앱의 번들 크기를 줄이기 위해 할 수 있는 모든 것](https://velog.io/@lky5697/reduce-webapp-bundle-size)
[[Korean FE Article] 자바스크립트 에코시스템의 속도 향상 - 배럴(Barrel) 파일의 대실패](https://github.com/yeonjuan/dev-blog/blob/master/JavaScript/speeding-up-the-javascript-ecosystem-the-barrel-file-debacle.md)
해당 글을 읽어봤는데, barrel export를 조금 섬세하게 써야할 것 같습니다.
어떻게 해야할까요?

**Comment by enigsuss** [comment]
저희 프로젝트의 규모가 아직 그정도로 큰 편은 아닌 것 같습니다. 
대신, 번들 사이즈 최적화 비교를 위해 기존 방식대로 작업 후 번들 사이즈를 측정한 뒤 
최적화 작업을 진행하면서 barrel export도 제거하는 방향으로 가는 것은 어떨까요?
